prime-router/src/main/kotlin/cli/ValidateTranslationSchemaCommand.kt

```diff
@@ -0,0 +1,73 @@
+package gov.cdc.prime.router.cli
+
+import com.github.ajalt.clikt.core.CliktCommand
+import com.github.ajalt.clikt.parameters.options.option
+import com.github.ajalt.clikt.parameters.options.required
+import com.github.ajalt.clikt.parameters.types.enum
+import gov.cdc.prime.router.azure.BlobAccess
+import gov.cdc.prime.router.fhirengine.translation.TranslationSchemaManager
+import org.apache.logging.log4j.kotlin.Logging
+
+class ValidateTranslationSchemaCommand :
+    CliktCommand(
+    name = "validateSchemas",
+    help = "Commands to validate translation schemas"
+),
+    Logging {
+
+    private val translationSchemaManager = TranslationSchemaManager()
+
+    /**
+     * The schema type to validate
+     */
+    private val schemaType: TranslationSchemaManager.SchemaType by option(
+        "--schema-type",
+        "-s",
+        help = "which schema type to process"
```

Comment:
(super nit) This help text is formatted differently than the other two, maybe "The schema type to validate against" or maybe just capitalize "which". Or do nothing, this is so unimportant 😆

Reply:
(less of a nit) This might also be an appropriate place to spell out which folder in the container is used for each schema type. It's visible in the code, but not sure if that's externalized through the CLI at all.

Alternatively, when schemas are not found, the error message could include the schema type directory.